### ISSUE TITLE
Why you didn't considering including these models?

### ISSUE THREAD

**Issue by luiscosio** [tunasoup/synthetic-image-detection#1]
@tunasoup, just curious why you didn't include:

https://github.com/grip-unina/DMimageDetection
https://github.com/grip-unina/GANimageDetection

**Comment by tunasoup** [comment]
The goal was to include a few example models, whose key ideas were different from each other.
For example, https://github.com/grip-unina/GANimageDetection is a modified version of one of the included models. The original was chosen as the the authors provided two sets of pre-trained weights, which allowed to study how heavier data augmentation affects the accuracy for images generated by unseen models.

Of course, any model with a compatible PyTorch implementation can be added by copying the model to the [detector](https://github.com/tunasoup/synthetic-image-detection/blob/main/detectors) folder as a package, implementing the Detector class, and modifying the [detection.py](https://github.com/tunasoup/synthetic-image-detection/blob/main/detection.py) file accordingly.
